src/components/Header/Header.js

```diff
@@ -125,6 +126,17 @@ function Header() {
                 <BalanceModule />
               </>
             )}
+            {account && (
+              <div
```

Comment:
Let´s give the parent div a `height: 100%`

So that the :focus background takes the whole height.
Also the popover content should display at the same line where the header ends, not in the middle.

https://user-images.githubusercontent.com/22663232/134428890-3149a51a-9249-487a-9594-e1c8b746d295.mov